.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
TBH I don't love pulling in editor specific config to our repos.

Reply:
I get that feeling a tiny bit, but wonder if the benefits outweigh the drawbacks. My overall aim is we talk about formatting changes as little as possible. This config makes it less likely for a developer to be surprised by linting failures on CI. 

I don't feel super strongly though. We can rip this out and let each dev sort their editor out by themselves.

Reply:
> TBH I don't love pulling in editor specific config to our repos.

I thought this too but noticed we also have a `.vscode` in the monorepo from a few years back: https://github.com/ethereum-optimism/optimism/pull/1109

If we do remove this, we can add `.vscode` to the gitignore then explain how to set this up in CONTRIBUTING. Can be VSCode only instructions for now that are expanded over time

Reply:
Sounds like a good compromise to me.

Reply:
LGTM, let's also delete this existing `.vscode/settings.json` that's added in this PR

Reply:
[b97eb4e](https://github.com/ethereum-optimism/superchain-registry/pull/105/commits/b97eb4e5e050f8028fa6414e59b24288ae089758)

Reply:
[a3e513b](https://github.com/ethereum-optimism/superchain-registry/pull/105/commits/a3e513b23c1823faed9b3fcee450fb9833167155)